packages/evo/src/agent-functions/executeScript.ts

```diff
@@ -106,7 +106,7 @@ export const executeScript: AgentFunction<AgentContext> = {
         },
         result: {
           type: "string",
-          description: "The name of the variable to store the result of the script"
+          description: "The name of the variable to store the result of the script (starts with '_' and contains letters)"
```

Comment:
So we want global variables to be prefixed with an underscore? What do other projects do? I'd be more in favor of a special character like `$` personally if it doesn't cause any problems.

Reply:
Since it's going into javascript we could use UPPERCASE?

Reply:
That works for me. It just has to be a valid JS variable name that isn't used for script function arg names. I don't know what other projects do, but this is a simple solution to the problem.

Reply:
we can do both $ and UPPERCASE

Reply:
this is done